lib/phraseapp-in-context-editor-ruby/delegate/i18n_delegate.rb

```diff
@@ -4,7 +4,7 @@ module PhraseApp
   module InContextEditor
     module Delegate
       class I18nDelegate < Base
-        attr_accessor :key, :display_key, :options, :api_client, :fallback_keys, :original_args
+        attr_accessor :key, :display_key
```

Comment:
`#phraseapp_delegate_for` no longer passes the options as the last entry in `original_args`. This is a behaviour change to `original_args` inside I18nDelegate. I removed the attr_accessor here because nothing tries to use the `original_args` outside the class. So it is clear that the difference in behaviour is irrelevant outside of this class.

The other removed attr_accessors were also unused or nil.